config/checkstyle.xml

```diff
@@ -33,12 +33,12 @@
         <!-- Requires a Mongodb copyright notice in each file.
           <pre>
             /*
-             * Copyright 2008-2014 MongoDB, Inc.
+             * Copyright 2008-present MongoDB, Inc.
              *
           </pre>
         -->
         <property name="format"
-                  value="^(//| \*) Copyright (\([cC]\) )?[\d]{4}(\-[\d]{4})? (MongoDB, Inc\.).*$" />
+                  value="^(//| \*) Copyright 2008-present MongoDB, Inc\.$" />
```

Comment:
I didn't read the drivers ticket as starting every file at 2008.

Reply:
I did, the copyright is for the project as a whole, source and docs. It also makes it much easier as well.

Reply:
Yeah, I see what you mean after re-rereading the ticket.  But it was interpreted differently in PyMongo: https://github.com/mongodb/mongo-python-driver/commit/507f954ed4f5778b2cc2d34fd57ff5525f4d2be0.  I'll ask Bernie to take a look.

Reply:
Confirmed that this way is fine.